docs/source/index.rst

```diff
@@ -7,39 +7,47 @@ Dialog Flow Framework Documentation
 
 *Date*: |today| *Version*: |version|
 
-The Dialog Flow Framework (`DFF`) is an open-source `Apache 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`_-licensed library
-developed for creating dialog systems.
+The Dialog Flow Framework (`DFF`) is an open-source, `Apache 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`_-licensed library
+that was developed specifically for creating dialog systems. `DFF` provides a comprehensive set of tools and resources for building conversational interfaces
+for a wide range of applications, including chatbots, virtual assistants, and other interactive systems.
+It allows developers to easily create and manage complex dialog flows, integrate with natural language processing (NLP) tools,
+and handle user input in a flexible and efficient manner. Additionally, the framework is highly customizable,
+allowing developers to easily adapt it to their specific needs and requirements. 
 
 Getting started
 ---------------
 
 Installation
 ~~~~~~~~~~~~
 
-`DFF` can be installed via ``pip``:
+`DFF` can be easily installed on your system using the ``pip`` package manager:
 
 .. code-block:: console
    
    pip install dff
 
-and supports Python 3.7 and newer. You can also download the library from `Github <https://github.com/deeppavlov/dialog_flow_framework>`_:
+This framework is compatible with Python 3.7 and newer versions.
+Note that if you are going to use one of the database backends, you will have to specify an extra or install the corresponding requirements yourself.
```

Comment:
I think we should add an example or a reference of specification of installation the corresponding requirement